tests/coafileTest.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+import re
+from coalib.settings.ConfigurationGathering import load_config_file
+
+
+class coafileTest(unittest.TestCase):
+
+    def setUp(self):
+        self.filepath = '.coafile'
+        self.commits_filepath = 'tests/commits.txt'
+        self.sections = load_config_file(self.filepath, None, True)
+
+    def test_commit_regex(self):
+        shortlog_regex = self.sections['commit']['shortlog_regex'].value
+        git_log = open(self.commits_filepath, 'r').read()
+        for message in git_log.splitlines():
+            self.assertIsNotNone(re.fullmatch(shortlog_regex, message),
+                                 msg='\"{0}\" does not match the '
```

Comment:
you could maybe print the `repr` version of the string, then you don't need the double quotes `"` at all and python escapes stuff probably if necessary.
You can enable this behaviour by using `{!r}`.
Up to you :)